__tests__/test1.js

```diff
@@ -1,19 +1,22 @@
 const Pet = require('../src/test1');
+// add a line break here
```

Comment:
You should always separate your imports from the rest of your code with a line break.